modules/subchannel/src/problems/SubChannel1PhaseProblem.C

```diff
@@ -473,12 +466,11 @@ SubChannel1PhaseProblem::computeMdot(int iblock)
   {
     for (unsigned int iz = first_node; iz < last_node + 1; iz++)
     {
-      auto dz = _z_grid[iz] - _z_grid[iz - 1];
       for (unsigned int i_ch = 0; i_ch < _n_channels; i_ch++)
       {
         auto * node_in = _subchannel_mesh.getChannelNode(i_ch, iz - 1);
         auto * node_out = _subchannel_mesh.getChannelNode(i_ch, iz);
-        auto volume = dz * (*_S_flow_soln)(node_in);
+        auto volume = _dz * (*_S_flow_soln)(node_in);
```

Comment:
no, why did you make this change?
this was nice flexibility to have a non uniform grid